dll/win32/netid/lang/ro-RO.rc

```diff
@@ -2,93 +2,93 @@
  * FILE:          dll/win32/netid/lang/ro-RO.rc
  *                ReactOS Project (https://reactos.org)
  * TRANSLATORS:   Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
- *                Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
+ *                Copyright 2023-2024 Andrei Miloiu <miloiuandrei@gmail.com>
  */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 IDD_PROPPAGECOMPUTER DIALOGEX 0, 0, 256, 218
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Numele calculatorului"
+CAPTION "Nume computer"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_COMPNAME, IDC_STATIC, 4, 3, 26, 24, SS_ICON
-    LTEXT "Informațiile de mai jos vor fi utilizate pentru identificarea calculatorului în rețea.", IDC_STATIC, 40, 5, 204, 20
-    LTEXT "Descrierea calculatorului:", IDC_STATIC, 7, 40, 90, 9
+    LTEXT "Pentru identificarea computerului în reţea, ReactOS utilizează următoarele informaţii.", IDC_STATIC, 40, 5, 204, 20
+    LTEXT "&Descriere computer:", IDC_STATIC, 7, 40, 90, 9
     EDITTEXT IDC_COMPDESC, 98, 38, 146, 13, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    LTEXT "Numele complet:", IDC_STATIC, 7, 68, 85, 9
+    LTEXT "Numele complet al computerului:", IDC_STATIC, 7, 68, 85, 9
```

Comment:
looks too long to my eyes for 85 width

Reply:
I checked with ReactOS's latest nightly build LiveCD (reactos-livecd-0.4.15-dev-8038-g28eb4e7-x86-gcc-lin-dbg) and it fits perfectly. I also checked with XP's dll and it also fits in the box. Here is the proof.

![2024-05-05 23_26_45-Resource Hacker - netid dll mui](https://github.com/reactos/reactos/assets/113553683/f97a3d5e-b8f3-4529-9fd3-bc3754167277)

![2024-05-05 23_24_33-](https://github.com/reactos/reactos/assets/113553683/761866ce-e9a6-4440-9a8b-5844ba7339e1)

Reply:
Thx. Resolved.